src/common/collection.c

```diff
@@ -1582,32 +1582,55 @@ static gchar *get_query_string(const dt_collection_properties_t property, const
       break;
 
     case DT_COLLECTION_PROP_TAG: // tag
-      if ((escaped_length > 0) && (escaped_text[escaped_length-1] == '*'))
+      if(!strcmp(dt_conf_get_string("plugins/lighttable/tagging/case_sensitivity"), "insensitive"))
```

Comment:
you need to get the string in a variable and free it after.

Reply:
You're right ...

Reply:
done in collect.c as well